j2/vars/custom-webiny-rules.yaml

```diff
@@ -1463,4 +1463,274 @@ rules:
 
       - label: Has_Data_Source
         fromVertexLabel: "Object_Contract"
-        toVertexName: "attached_data_sources"
\ No newline at end of file
+        toVertexName: "attached_data_sources"
+
+  webiny_organisation:
+
+    vertices:
+
+      - label: Person_Organisation
+        condition: "${organizacoesCnpj?.replaceAll('[^0-9]','')?.length() == 14}"

```

Comment:
No point in differentiating between international from national companies.  Please just add a condition of length()>0, and simplify the whole thing.